workshop/content/60-go/30-hello-cdk/100-cleanup.md

```diff
@@ -0,0 +1,90 @@
++++
+title = "Cleanup sample"
```

Comment:
Needs better title.

Reply:
I think this is pretty well descriptive of the contents of this page, what are you thinking of instead?